src/Casbin.AspNetCore.Core/ServiceCollectionExtension.cs

```diff
@@ -10,8 +10,15 @@ namespace Casbin.AspNetCore.Authorization
 {
     public static class CoreServiceCollectionExtension
     {
-        public static IServiceCollection AddCasbinAuthorizationCore(
-            this IServiceCollection services,
+        /// <summary>
+        /// Adds casbin core services to the specified <see cref="IServiceCollection" />
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        /// <param name="configure"></param>
+        /// <param name="defaultModelProviderLifeTime"></param>
+        /// <param name="defaultEnforcerProviderLifeTime"></param>
+        /// <returns></returns>
+        public static IServiceCollection AddCasbin(this IServiceCollection services,
             Action<CasbinAuthorizationOptions>? configure = default,
             ServiceLifetime defaultModelProviderLifeTime = ServiceLifetime.Scoped,
```

Comment:
Is there any mechanisim to ensure correct lifetime scope,as `DefaultEnforcerProvider` requires `DefaultModelProvider` injection, or it depends on users' consciousness?

Reply:
The 'DefaultEnforcerProvider' requires 'DefaultModelProvider' injection, So you will get an exception when starting the project if you set the singleton model and scope enforcer. It will work in other settings.

Reply:
yah.I think sth. like an enum for scope group defination maybe work.

Reply:
I don't think this is necessary, because in most cases, the default configuration should be used. And the user should know the relationship between model and enforcer.